model-optimizer/mo/front/kaldi/loader/loader.py

```diff
@@ -200,11 +200,13 @@ def load_kaldi_nnet3_model(graph, file_descr, nnet_name):
     # add information for shape calculation for MemoryOffset
     # shape calculation for MemoryOffset can't be done through shape of previous layer because
     # it is separated in 2 parts to remove cycle from graph
+    batch = graph.graph['cmd_params'].batch if graph.graph['cmd_params'].batch is not None else 1
```

Comment:
we also can have a --input_shape set (not together with --batch but as a separate case)

Reply:
fixed